test/unit/run-summary.test.js

```diff
@@ -48,6 +49,35 @@ describe('run summary', function () {
         expect(summary.run.transfers).to.be.an('object');
     });
 
+    it('should have empty executions array', function () {
+        var options = {
+                executionsSummary: false
```

Comment:
Add test for `executionsSummary`:
- false
- true
- undefined (default behavior, don't set `undefined` explicitly)

Reply:
And another set of tests for how it changes based on the `reporters` option.